src/index.html

```diff
@@ -8,6 +8,83 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <section class="stars stars--0">
+      <div class="stars__star">
+      </div>
+      <div class="stars__star">
+      </div>
+      <div class="stars__star">
+      </div>
+      <div class="stars__star">
+      </div>
+      <div class="stars__star">
+      </div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
```
fix code style all over the page